src/ProjectOrigin.WalletSystem.Server/Services/REST/v1/ModelsV1.cs

```diff
@@ -47,3 +47,18 @@ public record Claim
     public required ClaimedCertificate ProductionCertificate { get; init; }
     public required ClaimedCertificate ConsumptionCertificate { get; init; }
 }
+
+public record AggregatedCertificates
+{
+    public required long Start { get; init; }
+    public required long End { get; init; }
+    public required long Quantity { get; init; }
+    public required CertificateType Type { get; init; }
```

Comment:
What value does this have when queryin both consumption and production?

Reply:
it returns two records, one for production, and one for consumption. Since the sum of the two types doesn't make sense, it would be like saying i got 1000$ in salary and have expenses for 500$ so I have a sum of 1500$...